js/graph_spectrum_plot.js

```diff
@@ -1,16 +1,27 @@
 "use strict";
 
-const DEFAULT_FONT_FACE       = "Verdana, Arial, sans-serif",
+const BLUR_FILTER_PIXEL       = 1,
+      DEFAULT_FONT_FACE       = "Verdana, Arial, sans-serif",
       DEFAULT_MARK_LINE_WIDTH = 2,
-      MARGIN                  = 10; // pixels;
+      MARGIN                  = 10,
+      MARGIN_BOTTOM           = 10,
+      MARGIN_LEFT             = 25,
+      MARGIN_LEFT_FULLSCREEN  = 35;
+
+const SPECTRUM_TYPE = {
+        FREQUENCY        : 0,
+        FREQ_VS_THROTTLE : 1,
+      };
 
 var GraphSpectrumPlot = GraphSpectrumPlot || {
-    _isFullScreen : false,
-    _cachedCanvas : null,
-    _canvasCtx    : null,
-    _fftData      : null,
-    _mouseFrequency : null,
-    _sysConfig    : null,
+    _isFullScreen     : false,

```

Comment:
This is a good example of why I am generally not in favour of aligning code - reviewing this is made harder by the fact that it had to be realigned...

Reply:
Yes, I try to not do it, I understood the reasons the latest time, but I do it without thinking (at my real job we don't make revisions). 😁